src/runtime/bootstrap-lazy.ts

```diff
@@ -157,6 +157,14 @@ export const bootstrapLazy = (lazyBundles: d.LazyBundlesRuntimeData, options: d.
         }
       }
 
+      // if the component is formAssociated we need to set that on the host
+      // element so that it will be ready for `attachInternals` to be called on
+      // it later on
+      if (BUILD.shadowDom && BUILD.formAssociated && cmpMeta.$flags$ & CMP_FLAGS.formAssociated) {
+        // any way around this typecast?
+        (HostElement as any).formAssociated = true;
```

Comment:
I don't love this but this does seem to be necessary - setting `HostElement.formAssociated` is necessary for it to work correctly and unfortunately the typing does't want to let me set the property on there :/

Reply:
I don't think there is, unfortunately. We should figure out these types tho cuz it feels mega wonky

Reply:
Agreed, we probably just need to remove the comment here